FRAuth/FRAuth.xcodeproj/xcshareddata/xcschemes/FRAuth.xcscheme

```diff
@@ -57,6 +57,11 @@
                BlueprintName = "FRAuthTests"
                ReferencedContainer = "container:FRAuth.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "AA_07_AppIntegrityTest/test_01_test_app_integrity()">
```

Comment:
Why are we skipping this? Should that be committed

Reply:
The tests is written, it will only work in local box for now and feature is not available in AM , so when the AM is ready we will add this test back